test/syscalls/linux/dev.cc

```diff
@@ -146,7 +146,27 @@ TEST(DevTest, WriteDevFull) {
   EXPECT_THAT(WriteFd(fd.get(), "a", 1), SyscallFailsWithErrno(ENOSPC));
 }
 
+TEST(DevTest, ReadDevFuse) {
+  SKIP_IF(IsRunningWithVFS1());
+
+  const FileDescriptor fd =
+      ASSERT_NO_ERRNO_AND_VALUE(Open("/dev/fuse", O_RDONLY));
+  std::vector<char> buf(1);
+  EXPECT_THAT(ReadFd(fd.get(), buf.data(), sizeof(buf)), SyscallFailsWithErrno(ENOSYS));
+}
+
+TEST(DevTest, WriteDevFuse) {
+  SKIP_IF(IsRunningWithVFS1());
+
+  const FileDescriptor fd =
+      ASSERT_NO_ERRNO_AND_VALUE(Open("/dev/fuse", O_WRONLY));
+  const char* testStr = "test";
+  EXPECT_THAT(WriteFd(fd.get(), testStr, sizeof(testStr)), SyscallFailsWithErrno(ENOSYS));
+}
+
 TEST(DevTest, TTYExists) {
+  SKIP_IF(!IsRunningWithVFS1());
```

Comment:
I think we should still run this test on Linux, right?

```cpp
bool IsRunningWithVFS1() {
  const char* env = getenv(GVISOR_VFS);
  if (env == nullptr) {
    // If not set, it's running on Linux.
    return false;
  }
  return strcmp(env, "VFS1") == 0;
}
```

Reply:
Doesn't look like the `/dev/tty` device is implemented in VFS2 yet.

Reply:
Specifically, I was suggesting changing this line to:
```cpp
SKIP_IF(!IsRunningWithVFS1() && IsRunningOnGvisor());
```

This allows the test to run on VFS1 and Linux.

Reply:
Oh I see. Done.